content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -8,15 +8,14 @@ content_type: concept
 weight: 60
 aliases:
 - controller-metrics.md
+- monitoring.md
 ---
 
 <!-- overview -->
 
 System component metrics can give a better look into what is happening inside them. Metrics are particularly useful for building dashboards and alerts.
 
-Metrics in Kubernetes control plane are emitted in [prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/) and are human readable.
-
-
+Metrics in Kubernetes components are emitted in [prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/) and are human readable.
```

Comment:
nit: you could reword this to (eg)
```suggestion
Kubernetes components emit metrics in [prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/). This format is structured plain text, designed so that people and machines can
both read it.
```